api/src/main/java/org/pmiops/workbench/api/AuditController.java

```diff
@@ -94,9 +93,12 @@ private static String auditSql(String cdrProjectId, List<String> tableSuffixes)
         tableSuffixes.stream().map(s -> "'" + s + "'").collect(Collectors.joining(",")));
   }
 
-  @AuthorityRequired({Authority.AUDIT})
   @Override
-  public ResponseEntity<AuditBigQueryResponse> auditBigQuery() {
+  public ResponseEntity<AuditBigQueryResponse> auditBigQuery(Boolean isAppengineCron) {
```

Comment:
Where does isAppengineCron get injected from?

We might want to have an annotation for cron jobs, and mess with AuthInterceptor to use it... up to you.

Reply:
Yeah, sounds reasonable. I switched to keying this check off of a `cron` tag in swagger. I added a separate interceptor since it's a bit orthogonal to our normal authn flow.

Previously that value was just getting autopopulated from the Swagger spec (I had defined that header there).